Warehouse.Client/warehouse-client/src/app/app.component.ts

```diff
@@ -0,0 +1,21 @@
+import {Component} from '@angular/core';
+import {User} from './models/user';
+import {JwtHelperService} from "@auth0/angular-jwt";
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent {
+  userName: string = this.jwtService.decodeToken()?.UserName;
```

Comment:
You can move such functionality to some ProfileService, which will encapsulate all profile-related functionality instead of calling JwtService directly